README.md

```diff
@@ -1 +1,1369 @@
-Example ForgeRock Platform Scripts
+
+# <a id="top"></a>Dynamic Scripting in ForgeRock Platform
+
+Three of ForgeRock Identity Platform products, [Access Management](https://www.forgerock.com/platform/access-management) (AM), [Identity Management](https://www.forgerock.com/platform/identity-management) (IDM), and [Identity Gateway](https://www.forgerock.com/platform/identity-gateway) (IG), allow to extend their functionality with dynamically applied, context-specific scripts written in JavaScript and Groovy and evaluated during the run time.
```

Comment:
Note that neither JavaScript nor Groovy are available as scripting options in every case. Groovy is the most widely supported, but IG doesn't support JS, IDM Workflows and Connectors don't support JS, and JS is required for client-side scripting in AM. 

My only point in saying this is that it might be more accurate to drop "written in JavaScript and Groovy" here, and just say "scripts".

Reply:
Maybe an "or" then, as I thought mentioning the options upfront will set the expectations. As a "non-shell" scripts.

Reply:
I have something about languages in regard to individual products product in the summary > Languages.